articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
+
+In Intent Architect, choose to create a new application and select the `Intent Application Template Builder`.
+Give it the name of `MyApplicationTemplates`.
+
+![Application Template Builder Creation](images/my-app-templates-creation.png)
+
+When clicking on `Next` you will be presented with the following page. There is nothing to customize here so you can click on `Create` to begin your own application templates.
+
+![Application Template Builder Customization](images/my-app-templates-customize.png)
+
+Once the installation dialog is complete, click on `Close`.
+
+![Installation dialog](images/installation-dialog.png)
+
+Open the `App Templates` designer and click on `Create new pacakge`.
+
+![Create App Template Package](images/create-app-templates-package.png)
+
+If you find that the package is highlighted in red, click on it and complete the following fields:
+
+![Application Template Package Settings](images/app-template-pacakge-settings.png)
+
+| Name | Description |
+|-|-|
+| Version | You can give it a default value of `1.0.0`. **It needs to have 3 version components.** |
+| Display Name | Give it a display friendly name. Example: `My Standardized Application Templates` |
+| Supported Client Versions | Which versions of Intent Architect does this target? Click [here](xref:references.notations#version-ranges) to find out about the version scheme. Use the default of: `[3.0.0,4.0.0)` |
+
+Once this is specified, follow through with the following steps (as an example) on how to setup Groups, Components and Modules.
+
+<p><video style="max-width: 100%" muted="true" loop="true" autoplay="true" src="videos/create-group-components.mp4"></video></p>
+
+As per the video, create a Group called `Presentation` and a Component in it called `Angular` (give it the icon URL of `https://www.elemental.co.za/cms/resources/uploads/blog/89/05bd53609c4c.png`, you can click on the `...` button which will allow you to paste the image data in as well from the OS clipboard), then to finish it off, create a Module in the Component that is the same name as the actual Module that you wish to import into your application: `Intent.Angular`. Specify that Module to use version `3.0.0-beta.6`.
+
+As a further exercise, complete the design so that it looks like this:
+
+![Completed design example](images/app-template-design-complete.png)
+
+The Services Component uses this URL `https://image.shutterstock.com/image-vector/service-tool-icon-this-isolated-260nw-274711127.jpg`.
+
+The Modules being used are:
+ * Intent.Modelers.Services (3.0.2)
+ * Intent.Application.ServiceImplementations (3.0.3)
+
+Now you can run the Software Factory to see the following staging changes:
+
+![Software Factory Run 1](images/software-factory-run-1.png)
+
+Click on `Apply` and open up the newly generated solution in Visual Studio.
```

Comment:
As per other article, we should elaborate a bit more on how to open in Visual Studio, or better, have a small article which we reference which explains how to open things in Visual Studio.